package.json

```diff
@@ -98,6 +98,7 @@
   "dependencies": {
     "@inrupt/solid-client": "^1.25.0",
     "@inrupt/solid-client-authn-browser": "^1.13.0",
+    "@storybook/mdx2-csf": "^0.0.3",
```

Comment:
The commit message introducing this mentions it as a dev dependency, but it appears here as a prod dependency.

Reply:
That was a mistake, amended commit.